webpack.js

```diff
@@ -41,9 +42,10 @@ webpackConfig.module.rules = Object.values(webpackRules)
 webpackConfig.plugins.push(
 	// patch webdav/dist/request.js
 	new webpack.NormalModuleReplacementPlugin(
-		/request.js/,
+		/request(\.js)?/,
 		function(resource) {
-			if (resource.context.indexOf('webdav/dist') > -1) {
+			if (resource.context.indexOf('webdav') > -1) {
+				console.debug('Patched request for webdav', basename(resource.contextInfo.issuer))
```

Comment:
This doesn't seem to work on latest master, though :S I've put a direct throw Error into patchedRequest and it never gets thrown?